### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 13

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#285] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152319473-23c6107e-9836-4f52-870a-64f0f81e117a.png)

**Comment by LuginiAndrea** [comment]
Dimostriamo che rispetta le proprietà di una equivalenza:
Riflessiva: $\forall s \in \mathbf{R}, s^2 = s^2 \Longrightarrow sRs$
Simmetrica: Notiamo subito come, essendo un'uguaglianza, invertire l'ordine nell'uguaglianza non varia la correttezza o meno della stessa, quindi $rRs \Longrightarrow r^2 = s^2 \Longrightarrow s^2 = r^2 \Longrightarrow sRr$
Transitività: $rRs \Longrightarrow r^2 = s^2$, $sRc \Longrightarrow s^2 = c^2$, $rRs \wedge sRc \Longrightarrow r^2 = c^2 \Longrightarrow rRc$

Inoltre $r^2 = s^2 \Longrightarrow r = \pm s$. Quindi per ogni numero $r \in \mathbf{R}$, $r \wedge -r$ elevati al quadrato hanno lo stesso valore. Ne consegue che le classi di equivalenza sono $\mathbf{R}^+ \cup 0$
